dbcrossbar/Cargo.toml

```diff
@@ -85,4 +85,4 @@ url = "2.1.0"
 uuid = "1.4"
 walkdir = "2.2.9"
 # Locked because upgrading it breaks Google Cloud with date-parsing errors.
-yup-oauth2 = "= 6.1.0"
+yup-oauth2 = "= 8.3.0"
```

Comment:
Did you see the date-parsing errors at any point while working on this PR? I've been meaning to look into them for ages, because they've kept me from upgrade `yup-oauth2`.

Reply:
no date parsing errors observed

Reply:
Hmm, fun. I've been trying to upgrade that library for ages and it always fails a bunch tests with date parsing errors. I'll need to look into this. Thank you for the confirmation!